### PR TITLE
scan: remove showing [do-not-modify]

### LINKED ISSUE #1611: Do not show [do-not-modify] in msg
Let's show other information if `[do-not-modify]` exists in msg on `Msg` or `TxIndexPage`.

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1611

### CODE REVIEW

**Comment by evilpeach** [comment]
@tansawit No need to make an address to option.

**Review by evilpeach — CHANGES_REQUESTED** [review]
It would be nice to make a helper function like
```
let checkDoNotModify = (x) => x == "[do-not-modify]"
```

**Review by evilpeach — DISMISSED** [review]

File: scan/src/components/TxIndexPageTable.re [hunk 1]
```diff
@@ -189,20 +189,20 @@ let renderEditDataSource = (dataSource: TxSub.Msg.EditDataSource.t) => {
...
+    //     <TypeID.DataSource id={dataSource.id} />
+    //     <HSpacing size=Spacing.sm />
+    //     <Text value={dataSource.name} />
+    //   </div>
     // </div>
+    // <VSpacing size=Spacing.md />
+    // // <div className=Styles.topicContainer>
+    // //   <Text value="FEE" size=Text.Sm weight=Text.Thin spacing={Text.Em(0.06)} />
+    // //   <AmountRender coins={dataSource.fee} pos=AmountRender.TxIndex />
+    // // </div>
```
Comment (prin-r) [inline scan/src/components/TxIndexPageTable.re]: Please remove comments
If you want to leave comments, please add reason with todo

File: scan/src/components/TxIndexPageTable.re [hunk 2]
```diff
@@ -238,15 +238,15 @@ let renderEditOracleScript = (oracleScript: TxSub.Msg.EditOracleScript.t) => {
...
-    </div>
+    // <VSpacing size=Spacing.lg />
+    // <div className=Styles.topicContainer>
+    //   <Text value="NAME" size=Text.Sm weight=Text.Thin spacing={Text.Em(0.06)} />
+    //   <div className=Styles.hFlex>
+    //     <TypeID.OracleScript id={oracleScript.id} />
+    //     <HSpacing size=Spacing.sm />
+    //     <Text value={oracleScript.name} />
+    //   </div>
+    // </div>
```
Comment (prin-r) [inline scan/src/components/TxIndexPageTable.re:251]: Please remove comments
If you want to leave comments, please add reason with todo

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**